packages/language-server/__tests__/get-token.test.ts

```diff
@@ -45,14 +45,42 @@ test('CSS var', () => {
 test('token reference with curly braces', () => {
   const ctx = createContext()
   expect(getTokenFromPropValue(ctx, 'border', '{colors.gray.300}')).toMatchInlineSnapshot(`
```

Comment:
@astahmer this test was broken due to the backing regex maintaining state across invocation due to the `g` flag being used on the expression. Removed the `/g` flags from the expression.

see: https://stackoverflow.com/a/1520853/684869